po/bn.po

```diff
@@ -0,0 +1,608 @@
+msgstr "X11 উইন্ডো সিস্টেম"
+
+#: src/models/sockets.js:44
+msgid "Wayland windowing system"
+msgstr "ওয়েল্যান্ড উইন্ডো সিস্টেম"
+
+#: src/models/sockets.js:51
+msgid "Fallback to X11 windowing system"
+msgstr "X11 উইন্ডো সিস্টেমে ব্যাকআপ"
+
+#: src/models/sockets.js:58
+msgid "PulseAudio sound server"
+msgstr "পালসঅডিও সাউন্ড সার্ভার"
+
+#: src/models/sockets.js:65
+msgid "D-Bus session bus"
+msgstr "ডি-বাস সেশন বাস"
+
+#: src/models/sockets.js:72
+msgid "D-Bus system bus"
+msgstr "ডি-বাস সিস্টেম বাস"
+
+#: src/models/sockets.js:79
+msgid "Secure Shell agent"
+msgstr "সিকিউর শেল এজেন্ট"
+
+#: src/models/sockets.js:86
+msgid "Smart cards"
+msgstr "স্মার্ট কার্ডসমূহ"
+
+#: src/models/sockets.js:93
+msgid "Printing system"
+msgstr "প্রিন্টিং সিস্টেম"
+
+#: src/models/sockets.js:100
+msgid "GPG-Agent directories"
+msgstr "জিপিজি-এজেন্ট ডিরেক্টরি"
+
+#: src/models/sockets.js:107
+msgid "Inherit Wayland socket"
+msgstr "ওয়েল্যান্ড সকেট উত্তরাধিকারসূত্রে পাওয়া"
+
+#: src/models/sockets.js:132
+msgid "List of well-known sockets available in the sandbox"
+msgstr "স্যান্ডবক্সে উপলব্ধ সুপরিচিত সকেটগুলির তালিকা"
+
+#: src/models/variables.js:40
+msgid "Variables"
+msgstr "ভেরিয়েবলস"
+
+#: src/models/variables.js:43
+msgid "e.g. GTK_DEBUG=interactive"
+msgstr "যেমন GTK_DEBUG=interactive"
+
+#: src/models/variables.js:73
+msgid "List of variables exported to the application"
+msgstr "অ্যাপ্লিকেশনে রপ্তানিকৃত ভেরিয়েবলগুলির তালিকা"
+
+#. TRANSLATORS: <full-month-name> <day-of-month>, <year-with-century>
+#: src/widgets/appInfoViewer.js:61
+msgid "%B %e, %Y"
+msgstr "%B %e, %Y"
+
+#: src/widgets/appInfoViewer.ui:46
+msgid "Version"
+msgstr "সংস্করণ"
+
+#: src/widgets/appInfoViewer.ui:59
+msgid "Last Updated"
+msgstr "সর্বশেষ আপডেট"
+
+#: src/widgets/appInfoViewer.ui:72
+msgid "Runtime"
+msgstr "রানটাইম"
+
+#: src/widgets/busNameRow.js:35 src/widgets/pathRow.js:73
+#: src/widgets/relativePathRow.js:33 src/widgets/variableRow.js:33
+msgid "This is not a valid option"
+msgstr "এটি একটি বৈধ বিকল্প নয়"
+
+#: src/widgets/detailsButton.js:60
+msgid "_Show Details"
+msgstr "_বিস্তারিত দেখুন"
+
+#: src/widgets/detailsButton.js:107
+msgid "Show application in a software manager"
+msgstr "সফটওয়্যার ম্যানেজারে অ্যাপ্লিকেশন দেখান"
+
+#: src/widgets/detailsButton.js:109
+msgid "No software manager found"
+msgstr "কোনও সফটওয়্যার ম্যানেজার পাওয়া যায়নি"
+
+#: src/widgets/globalRow.ui:7 src/widgets/globalInfoViewer.ui:21
+msgid "All Applications"
+msgstr "সমস্ত অ্যাপ্লিকেশন"
+
+#: src/widgets/globalInfoViewer.ui:32
+msgid "Changes that apply to all Flatpak applications"
+msgstr "সমস্ত ফ্ল্যাটপ্যাক অ্যাপ্লিকেশনে প্রযোজ্য পরিবর্তনসমূহ"
+
+#: src/widgets/globalInfoViewer.ui:55
+msgid "Flatpak Version"
+msgstr "ফ্ল্যাটপ্যাক সংস্করণ"
+
+#: src/widgets/globalInfoViewer.ui:68
+msgid "Portal Version"
+msgstr "পোর্টাল সংস্করণ"
+
+#: src/widgets/globalInfoViewer.ui:81
+msgid "Changes Path"
+msgstr "পথ পরিবর্তন"
+
+#: src/widgets/menu.ui:7
+msgid "_Help"
+msgstr "_সাহায্য"
```

Comment:
I wonder if the mnemonic will work here, e.g., I noticed gnome-shell Bengali translation simply ignores the `_` prefix. Maybe we should also do the same here.

Reply:
I can leave that in english if. You want I don't know that gnome shell ignore that `_`

Reply:
Not sure either, but I would not suggest to leave it in English. Perhaps just remove the `_` prefix it, and change all other similar strings to keep it consistent.